doc/source/whatsnew/v2.3.0.rst

```diff
@@ -48,6 +48,8 @@ These are bug fixes that might have notable behavior changes.
 
 .. _whatsnew_230.notable_bug_fixes.notable_bug_fix1:
 
+- The ``to_latex`` function is now fixed for hidden multiindex dataframes (:issue:`52218`)
```

Comment:
Not sure this is the right place for this, and its a bit misleading as well. I would write.

...`to_latex` styling of column headers when combined with hidden indexes or hidden index-level is fixed...

Reply:
@attack68 Ah didn't see the general bug fixes section... moved and reworded 👍